CMakeLists.txt

```diff
@@ -110,8 +110,13 @@ check_type_size ( "unsigned short" SIZEOF_UNSIGNED_SHORT )
 
 # Check for printf "%zu" size_t formatting support.
 if(CMAKE_CROSSCOMPILING)
-  set(HAVE_PRINTF_Z ON)
-  message (STATUS "Cross compiling - assuming printf \"%zu\" size_t formatting support")
+  if(WIN32)
```

Comment:
Is there a reason why the 'check_c_source_runs(...)' check below can't automatically handle this? I suspect that cmake can't do "runs" tests when cross-compliling... is that correct?